extensions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -127,23 +72,35 @@ func CreateRKE1Snapshot(client *rancher.Client, clusterName string) error {
 			if snapshotObj.State != active {
 				return false, nil
 			}
+
+			snapshotTime, err := time.Parse(time.RFC3339, snapshot.Created)
```

Comment:
Missed this before, this is OS agnostic, correct? If I were to run a snapshot test that utilizes this `time.RFC3339` on macOS, Linux, Windows, I will get the same result, right?

Reply:
why is the snapshotTime required here? is it to get the latest one?

Reply:
Correct

Reply:
@anupama2501 yeah, it's for latest snapshot. There has been flakiness in the past with the incorrect snapshot being grabbed. This becomes an issue at the end of the test as resources are deleted and the wrong snapshot causes those resources to not exist.